docs/team/fsgmhoward.md

```diff
@@ -0,0 +1,81 @@
+# ZH Liu — Project Portfolio Page
+
+## Overview
+Patient Manager is a _Command Line Interface_ (CLI) application for **_general practitioners_** (GP)
+who work in polyclinics to manage their patient list.
+
+My teammate, together with me, developed this application as a team project for CS2113T module of NUS.
```

Comment:
Minor grammar check here - should be "I, together with my teammates..."

Reply:
Thank you! This has been resolved in the latest commit.